### ISSUE TITLE
我的是5个点，那么计算loss的时候这个行代码要修改吗？lkpt += kpt_loss_factor * ((1 - torch.exp(-d / (s*(4 * sigmas ** 2) + 1e-9 ))) * kpt_mask).mean()

### ISSUE THREAD

**Issue by 1095788063** [xialuxi/yolov7-landmark#5]
![3](https://user-images.githubusercontent.com/29217820/191538481-8da2a330-0008-44e1-9978-f3541f6e77e4.JPG)

`                            d = (pkpt_x - selected_tkpt[:, 0::3]) ** 2 + (pkpt_y - selected_tkpt[:, 1::3]) ** 2
                            s = torch.prod(selected_tbox[:, -2:], dim=1, keepdim=True)
                            kpt_loss_factor = (torch.sum(kpt_mask != 0) + torch.sum(kpt_mask == 0)) / torch.sum(kpt_mask != 0)

                            lkpt += kpt_loss_factor * ((1 - torch.exp(-d / (s*(4 * sigmas ** 2) + 1e-9 ))) * kpt_mask).mean()
                            #lkpt += (d * kpt_mask).sum() / torch.sum(kpt_mask != 0)`
